buildahimage/Dockerfile.git

```diff
@@ -0,0 +1,51 @@
+# Dockerfile.stable
+#
+# Build a Buildah image from the latest stable
+# version of Buildah on the Fedoras Updates System.
+# This builds a secured container that can run
+# safely with privileges within the container.
+#
+FROM fedora
+ENV GOPATH=/root/buildah
+
+# Install the softwware required to build Buildah,
+# Then create a directory and clone from the 
+# Buildah GitHub repository, then make and install
+# Buildah to the container.
+RUN dnf -y install --enablerepo=updates-testing \
+     make \
+     golang \
+     bats \
+     btrfs-progs-devel \
+     device-mapper-devel \
+     glib2-devel \
+     gpgme-devel \
+     libassuan-devel \
+     libseccomp-devel \
+     ostree-devel \
+     git \
+     bzip2 \
+     go-md2man \
+     runc \
+     fuse-overlayfs \
+     fuse3 \
+     containers-common; \
+     mkdir /root/buildah; \
+     git clone https://github.com/containers/buildah /root/buildah/src/github.com/containers/buildah; \
+     cd /root/buildah/src/github.com/containers/buildah; \
+     make;\
+     make install
```

Comment:
A few thoughts on the RUN command above:
- Can we split the the package installation and the git clone + build into separate runs? The thought is that it's a pretty long running RUN task and network flakes after package installation would allow for reusing the package-installation layer.
- We could/should remove `/root/buildah` after `make install` to reduce the image size. My local repo with some remotes is ~65M.